### PR TITLE
NEW hook on ics generation to add more events in eventarray

### PR DESCRIPTION
**Pull request by atm-greg**
# New hook to add more events in actioncomm::build_exportfile process
This hook gives the ability to add more events by overwriting the eventArray before ICS generation.

### CODE REVIEW

File: htdocs/comm/action/class/actioncomm.class.php [hunk 1]
```diff
@@ -1721,6 +1721,13 @@ public function build_exportfile($format, $type, $cachedelay, $filename, $filter
                     }
                     $diff++;
                 }
+
+				$parameters=array('filters' => $filters, 'eventarray' => $eventarray);
```
Comment (eldy) [inline htdocs/comm/action/class/actioncomm.class.php]: What do you think of changing
$parameters=array('filters' => $filters, 'eventarray' => $eventarray);
into
$parameters=array('filters' => $filters, 'eventarray' => &$eventarray);

$eventarray can be very large. But i am not sure it works

Comment (atm-greg) [reply htdocs/comm/action/class/actioncomm.class.php]: I thought it was a good idea but to respect what is done everywhere else I decided not to do that.
I fix it now.

**Comment by atm-greg** [comment]
ping @eldy I've tested with the "&" and it works fine too

**Comment by atm-greg** [comment]
ping @eldy

**Pull request merged**